test/WhitelistedCommitment.js

```diff
@@ -726,15 +726,64 @@ contract(
       });
     });
 
-    describe("successful comittment", () => {
+    describe("successful commitment", () => {
       it("should burn unused neumarks from fixed pool", async () => {
         // sets min cap so commitment is successful
         await shouldBurnUnusedNeumarks(etherToWei(0.5));
       });
-    });
 
-    describe("finalized comittment", () => {
-      it("should not reclaim Neumark token before finalization");
+      it("should not reclaim Neumark token before finalization", async () => {
+        const { commitment, neumark } = await deployAllContracts(
+          lockAdminAccount,
+          whitelistAdminAccount
+        );
+
+        await expect(commitment.reclaim(neumark.address)).to.revert;
+      });
+
+      it("should not allow neumark trading", async () => {
+        const startingDate = closeFutureDate();
+        const duration = MONTH;
+        const mutableCurve = await deployMutableCurve();
+        const investor1 = accounts[0];
+        const investor2 = accounts[1];
+        const fixedInvestors = [investor1];
+        const fixedDeclaredTickets = [etherToWei(1.21981798)];
+        const actualInvestor1Commitment = etherToWei(1.21981798);
+        const expectedNeumarkAmmount = (await mutableCurve.issueInEth(
+          actualInvestor1Commitment
+        ))
+          .div(2)
+          .round(0, 4);
+
+        const {
+          commitment,
+          neumark,
+          lockedAccount
+        } = await deployAllContracts(lockAdminAccount, whitelistAdminAccount, {
+          commitmentCfg: {
+            fixedInvestors,
+            fixedTickets: fixedDeclaredTickets,
+            startTimestamp: startingDate,
+            duration
+          }
+        });
+        await setTimeTo(startingDate);
+        await commitment.commit({
+          value: actualInvestor1Commitment,
+          from: investor1
+        });
+
+        await setTimeTo(startingDate + duration + HOUR);
+
```

Comment:
where do you finalize commitment? the point of this test is to make sure that Neumarks are not transferable after finalization. Which differs for the public commitment which enables Neumarks